web/app/index.html

```diff
@@ -39,6 +40,8 @@
     <script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
     <script src="bower_components/lodash/lodash.js"></script>
     <script src="bower_components/angular-uuid/uuid.min.js"></script>
+    <script src="bower_components/leaflet/dist/leaflet.js"></script>
+    <script src="bower_components/leaflet/dist/leaflet-src.js"></script>
```

Comment:
Hmm, good point -- this is just what happened when I did `bower install` but it's a bit strange to have both. I'll try taking out the minified version and see if that breaks anything.